src/main/java/uk/gov/companieshouse/api/accounts/utility/filetransfer/FileTransferToolImpl.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.companieshouse.api.accounts.utility.filetransfer;
+
+import com.amazonaws.SdkClientException;
+import com.amazonaws.services.s3.AmazonS3;
+import com.amazonaws.services.s3.model.GetObjectRequest;
+import com.amazonaws.services.s3.model.S3Object;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import org.apache.commons.io.IOUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.util.StringUtils;
+import uk.gov.companieshouse.api.accounts.CompanyAccountsApplication;
+import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.logging.Logger;
+import uk.gov.companieshouse.logging.LoggerFactory;
+
+@Component
+public class FileTransferToolImpl implements FileTransferTool {
+
+    private static final Logger LOGGER = LoggerFactory
+        .getLogger(CompanyAccountsApplication.APPLICATION_NAME_SPACE);
+    private static final String S3_BUCKET = "s3://";
+    private static final String PATH_DELIMITER = "/";
+
+    private final String awsAccessKeyId;
+    private final String awsSecretAccessKey;
+    private final String imageCloudProxyHost;
+    private final Integer imageCloudProxyPort;
+
+    private final AmazonS3Handler amazonS3Handler;
+    private final EnvironmentReader environmentReader;
+
+    @Autowired
+    FileTransferToolImpl(EnvironmentReader environmentReader, AmazonS3Handler amazonS3Handler) {
+        this.environmentReader = environmentReader;
+        this.amazonS3Handler = amazonS3Handler;
+
+        awsAccessKeyId = environmentReader.getMandatoryString("AWS_ACCESS_KEY_ID");
```

Comment:
you shouldnt have to supply the access key and secret. Basic credentials are not required.